### ISSUE TITLE
Cannot use 'surf' browser 

### ISSUE THREAD

**Issue by amzon-ex** [microsoft/wslg#595]
I'm trying to use the [surf browser](https://surf.suckless.org/) on WSL and after installing when I run `surf` on the terminal, surf does not run.

# Environment

```none
Windows build number: 22000.348
Your Distribution version: [Ubuntu 20.04
Your WSLg version: 1.0.26
```

#  Expected behavior

A standalone browser window opens as expected.

# Actual behavior

The following error is thrown:
```none
WaylandCompositor requires eglBindWaylandDisplayWL, eglUnbindWaylandDisplayWL and eglQueryWaylandBuffer.
Nested Wayland compositor could not initialize EGL

(surf:2353): Gdk-WARNING **: 17:09:32.177: ../../../../../gdk/x11/gdkwindow-x11.c:5633 drawable is not a native X11 window
X Error of failed request:  BadWindow (invalid Window parameter)
  Major opcode of failed request:  18 (X_ChangeProperty)
  Resource id in failed request:  0x0
  Serial number of failed request:  10
  Current serial number in output stream:  11
```

**Comment by Biswa96** [comment]
Agree. I see the same error message. Workaround, set `GDK_BACKEND=x11` environment variable globally.

**Comment by amzon-ex** [comment]
@Biswa96 
Works for me, thank you. Could you kindly explain a little bit more about why the problem occurs, and if possible, insights into a more permanent fix in the future?

**Comment by Biswa96** [comment]
I wish I could can explain. I am not familiar with Linux GUI stack and eager to learn about it.

**Comment by ShadowyMist** [comment]
> Agree. I see the same error message. Workaround, set `GDK_BACKEND=x11` environment variable globally.
How do I set it globally?

**Comment by ophers** [comment]
> Agree. I see the same error message. Workaround, set `GDK_BACKEND=x11` environment variable globally.
> > How do I set it globally?

Simple:
```cmd
setx WSLENV GDK_BACKEND
setx GDK_BACKEND x11
```

**Comment by rubeng0nzalez** [comment]
> Agree. I see the same error message. Workaround, set `GDK_BACKEND=x11` environment variable globally.

Excellent solution!... ;)
